docs/pbc/all/search/202212.0/third-party-integrations/configure-algolia.md

```diff
@@ -0,0 +1,116 @@
+5. On the Algolia website, go to **Settings**. 
+6. Under **Team and Access**, click **API keys**.
+
+![algolia-keys](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-keys.png)
+
+7. From the **Your API Keys** tab, copy the following keys:
+    - Application ID
+    - Search-Only API Key
+    - Admin API Key
+8. Go back to your store's Back Office, to the Algolia app details page.
+9. In the top right corner of the Algolia app details page, click **Configure**.
+10. In the **Configure** pane, fill in the **APPLICATION ID**, **SEARCH-ONLY API KEY**, and **ADMIN API KEY** fields with the values from step 6.
+
+![algolia-settings](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-settings.png)
+
+11. Click **Save**. The Algolia app is now integrated into your store and starts exporting your product data automatically.
+
+{% info_block infoBox "Info" %}
+
+You need to wait for a few minutes until Algolia finishes the product export.
+The more products you have, the longer you have to wait. 
+The average export speed is around *100 products per minute*.
+
+{% endinfo_block %}
+
+<figure class="video_container">
+    <video width="100%" height="auto" controls>
+    <source src="https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-integration.mp4" type="video/mp4">
+  </video>
+</figure>
+
+
+{% info_block warningBox "Verification" %}
+
+Verify that your index is populated with data from your store:
+1. Go back to the Algolia website. 
+2. In the side pane, go to **Search&nbsp;<span aria-label="and then">></span> Index**. 
+3. Make sure that the index is now populated with data from your store.
+
+{% endinfo_block %}
+
+![algolia-index-data](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-index-data.png)
+
+For details about the created index data, see [Indexes](/docs/pbc/all/search/{{page.version}}/third-party-integrations/algolia.html#indexes).
+
+## Optional: Adjust Algolia configuration
+
+The default Algolia app configuration mimics the default Spryker search configuration. However, you may want to adjust some of those settings to your needs.
+
+### Overview of searchable attributes
+
+![algolia-searchable-attributes](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/third-party-integrations/configure-algolia/algolia-searchable-attributes.png)
+
+Algolia's **Searchable attributes** configuration determines which attributes are used to find results while searching with a search query.
+
+Default fields for searchable attributes are the following:
+- `sku`
+- `name`
+- `description`
+- `keywords`
+
+### Adjust the searchable attributes list
+
+1. In the side pane, go to **Search&nbsp;<span aria-label="and then">></span> Index**. 
+2. Open the Algolia indices list and find all primary indices. 
```

Comment:
I don't get this step. Where can I find this list? Looks like it is redundant.

Reply:
This step is done in Algolia.